crates/nu-cli/src/commands/alias.rs

```diff
@@ -93,10 +96,14 @@ pub async fn alias(
         let right_brace = raw_input.rfind('}').unwrap_or_else(|| raw_input.len());
         let left = raw_input[..left_brace]
             .replace("--save", "")
-            .replace("-s", "");
+            .replace("-si", "-i")
+            .replace("-s ", "")
+            .replace("-is", "-i");
         let right = raw_input[right_brace..]
             .replace("--save", "")
-            .replace("-s", "");
+            .replace("-si", "-i")
+            .replace("-s ", "")
+            .replace("-is", "-i");
```

Comment:
Oooof, this will get tough to manage if we ever wanted to add more flags (although, I understand `--infer` is temporary). Just want to point out that we should consider a save mechanism that doesn't need to do this.

/cc @jonathandturner since you've done a bunch of work on aliases

Reply:
Thanks for the heads up. Yeah, I'll try to keep this in mind as we improve alias further.